requests-for-comments/2023-17-produchog.md

```diff
@@ -0,0 +1,92 @@
+
+If not Substack, we’ll move to [Customer.io](http://Customer.io) as originally planned.
+
+### Who is ProductHog for?
+
+As is currently the case, it’ll be closely aligned to our ICP and target in order of importance:
+
+- CEOs and founders
+- Product engineers
+- Product managers and product owners
+- Growth hackers and product marketers
+
+### Who is ProductHog NOT for?
+
+- Indie hackers (they may find it useful, but it won’t be tailored to their interests)
+- Marketing teams (even when we discuss marketing adjacent topics)
+- Traditional “unengaged” engineers
+- Data scientists
+
+## How to 10x ProductHog
+
+### Make each issue more thematic
+
+Currently, each issue is a bit of a mishmash of different topics and ideas. This provides variety, but not much focus. Curation should still be a core part of the newsletter, but ideally each issue should be more like a magazine, where the core content is themed around a particular topic or idea.
+
+### Double down on curation
+
+In addition to more focused original content, we should become known as a leading source for curated articles on product growth and high-growth startups. This means having more curated articles and (likely) adding categories to organise them more clearly for readers.
+
+### De-emphasise PostHog specific tutorials
+
+Having a high emphasis on content about doing specific things in PostHog will put off people who aren’t already PostHog users. Our goal with ProductHog is to raise awareness of PostHog and promote our values, rather than inform existing users of new content / how to stuff. We should launch a new email list / create a new way to alert PostHog users to new tutorials.
+
+### Introduce a regular feature(s)
+
+Open to suggestions here, but I was thinking we could do something around highlighting an interesting company / product each issue. It would help us with mutual promotion of the newsletter. Could be primarily focused on users of PostHog, but not exclusively. This would have to be freelanced as an ongoing commission to reduce load. 
+
+**Challenge:** How to avoid this becoming cookie-cutter, boring marketing guff – i.e. interview with so-and-so about “something only they care about”?
+
+### Introduce guest contributors
+
+I’d like to start commissioning guest contributors from well-known and successful newsletters in our space, either to write pieces for us or collaborate on articles with James on topics of mutual interest to our audiences. This might get expensive, but would be more valuable than a simple sponsorship.
```

Comment:
i'd happily do some kind of interviews, as it'd allow me to ask stranger / deeper / more specific questions than 99% of people tend to write. 

would _we_ really put the extra effort into a piece of content on someone else's newsletter with relatively low distribution?

Reply:
Cover some YC companies and founders relevant to our audience?

Reply:
I like the 60 Seconds With... style of interview. A set of simple, set questions we share with a bunch of people to make this quick, easy, fun, and scalable.

Reply:
> I like the 60 Seconds With... style of interview. A set of simple, set questions we share with a bunch of people to make this quick, easy, fun, and scalable.

This was my thinking

Reply:
This is a great way to deal with the curation issue on Substack (ie. the fact that all the top newsletters post original content - a few do the whole 'here's the original content, and then some links to other things afterwards.')

I'd probably book in a 15min call to ask them the questions then write them up yourself though? I think the friction of writing carefully curated answers might put people off. But agree interview transcripts are horrible. 

Random example questions of varying quality:
- What important product 'truth' do very few people agree with you on? (adapted from Peter Thiel)
- When have you been lucky?
- Why does your product matter?
- What is your favorite product that no one is talking about?
- What was the last product you recommended to someone?
- Pineapple on pizza?

Reply:
genuine q - do we think anyone will care about unknown YC founders?

Reply:
Honestly, no idea, but I'm moving a bit away from this atm anyway.